_docs-sources/foundations/landing-zone/manage-accounts.md

```diff
@@ -35,40 +35,33 @@ Below you'll find a table with common AWS account operations and the Gruntwork r
 
+Note that we are recommending that ultimately accounts be closed with ClickOps instead of using Gruntwork Pipelines. Removing the account via pipelines by deleting the account request file can and often does work, however the underlying AWS Service Catalog that we use to interact with Control Tower and deprovision the account is not reliable and often returns spurious errors that can require multiple retries to complete successfully.  The procedure here is fundamentally about working around that unreliability.
 
+#### 1. Cleanup the OpenTofu/Terraform State for the Control Tower module
 
-#### 1. (Optional) Destroy resources in the account to be removed
-
-You may choose or need to destroy some or all of your provisioned resources, if the resources are managed by Terragrunt in your infrastructure repository which has Gruntwork Pipelines integrated, you can destroy them by doing the following:
-
-1. Checkout a new branch in the infrastructure repository
-1. Remove the folder(s) containing the resources you want to destroy
-1. Commit your changes and create a Pull Request which will trigger Gruntwork Pipelines to destroy the resource once the Pull Request is merged.
-
-    :::danger
-    Make sure you are deleting the correct resources in the right account and that you have a backup of any data you may need in the future.
-
-    It is not recommended that you delete the AWS IAM OIDC provider or the AWS IAM roles used by Pipelines within the AWS account before closing the account. This ensures that you can continue to use Pipelines to manage resources within it (including destroying resources) prior to its full closure.
-    :::
+1. Obtain AWS CLI credentials for the your *management* account.
+1. Remove the targeted account(s) from being controlled by the *management* account's Control Tower module by running the following commands:
 
-    :::note
-    You may also need to delete resources if you encounter issues with coordination between different accounts where resources remain present in the account to be closed that are referenced in other accounts.
-    :::
+  ```bash
+  cd management/_global/control-tower-multi-account-factory/ # Navigate to the Control Tower module directory in your central/root infrastructure repository
+  terragrunt plan # Observe that targeted accounts are set for destruction
+  terragrunt state rm 'module.accounts["<ACCOUNT_NAME>"].aws_servicecatalog_provisioned_product.control_tower_factory' # Optionally, use 'rm -dry-run' to preview the removal.
+  terragrunt plan # Verify no accounts are set for destruction but outputs are updated
+  terragrunt apply # Apply the changes.
+  ```
 
+#### 2. Cleanup Infrastructure Code
 
-#### 2. Cleanup Central/Root Infrastructure Repository
+All code changes should be committed for each repository, **at the end**, to Git with a **[skip ci]** in the commit message to avoid triggering Gruntwork Pipelines CI processes.
```

Comment:
```suggestion
All the code changes recommended here should be committed for each repository, **at the end**, to Git with a **[skip ci]** in the commit message to avoid triggering Gruntwork Pipelines CI processes.
```